#! /bin/sh /usr/share/dpatch/dpatch-run
## 20_makefile_fixup.dpatch by Roland Rosenfeld <roland@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix syntax error in GNUmakefile.in

@DPATCH@
diff -urNad privoxy~/GNUmakefile.in privoxy/GNUmakefile.in
--- privoxy~/GNUmakefile.in
+++ privoxy/GNUmakefile.in
@@ -672,9 +672,9 @@
 	$(RM) -r doc/source/user-manual/
 	mkdir -p doc/text doc/source/user-manual
 	cd doc/source/user-manual && $(DB) -iuser-man ../user-manual.sgml && cd .. && cp user-manual/*.html ../webserver/user-manual/
-	@#FIXME: temp fix so same stylesheet gets in more than one place so it works \
-	@# for all doc set-ups, including the 'user manual' config option in local \
-	@#system where it MUST be in same directory as html.
+	#FIXME: temp fix so same stylesheet gets in more than one place so it works \
+	# for all doc set-ups, including the 'user manual' config option in local \
+	#system where it MUST be in same directory as html.
 	$(PERL) -pi.bak -e 's/<\/head/\n<LINK REL=\"STYLESHEET\" TYPE=\"text\/css\" HREF=\"p_doc.css\">\n<\/head/i' doc/webserver/user-manual/*html
 	cd doc/source && $(DB) -iuser-man -V nochunks user-manual.sgml > tmp.html && $(WDUMP) tmp.html > ../text/user-manual.txt && $(RM) -r tmp.html user-manual
 
